operators/postgresql-operator/PostgresqlAgent.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+const CONST = require('../../common/constants');
+const Agent = require('../../data-access-layer/service-agent');
+const logger = require('../../common/logger');
+
+class PostgresqlAgent extends Agent {
+  constructor(settings) {
+    super(settings);
+  }
+
+  createDb(ips, instanceId, params) {
+    const body = {
+      parameters: params
+    };
+    return this
+      .getHost(ips, CONST.AGENT.FEATURE.MULTI_TENANCY)
+      .tap(ip => logger.debug(`Ip to handle logicaldb creation request, ${ip}`))
+      .then(ip => this.post(ip, `tenants/${instanceId}`, body, 200));
```

Comment:
use `constant.HTTP_STATUS_CODE.OK` instead of 200 here.

Reply:
Done.